dev/com.ibm.websphere.appserver.features/visibility/auto/io.openliberty.restfulWS3.0-jaxb3.0.feature

```diff
@@ -2,7 +2,7 @@
 symbolicName=io.openliberty.restfulWS3.0-jaxb3.0
 visibility=private
 IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.restfulWS-3.0))", \
- osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.jaxb-3.0))"
+ osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=io.openliberty.xmlBinding-3.0))"
```

Comment:
Should we change the feature name as well from jaxb to xmlBinding?

Reply:
Sure